lib/grant-types/client-credentials-grant-type.js

```diff
@@ -54,7 +54,7 @@ ClientCredentialsGrantType.prototype.handle = function(request, client) {
     throw new InvalidArgumentError('Missing parameter: `client`');
   }
 
-  var scope = this.getScope(request);
+  const scope = this.getScope(request);
```

Comment:
This should be `let scope = ...`

Reply:
Can you please elaborate? I can't find any reassignment to this variable and the tests did not fail. If this is likely to produce an error we need to update the tests definitely.

Reply:
I say *may* be problematic because `scope` is passed back out the models so it could cause issues for some if is suddenly immutable. I don't think we need to update the test.

Reply:
I see. We can change it to `let`